crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/gossip.rs

```diff
@@ -19,7 +20,7 @@ ghost_actor::ghost_chan! {
             dht_arc: DhtArc,
             since_utc_epoch_s: i64,
             until_utc_epoch_s: i64,
-        ) -> Vec<Arc<KitsuneOpHash>>;
+        ) -> (Vec<Arc<KitsuneOpHash>>, Vec<AgentInfoSigned>);
```

Comment:
Hmmm - I'm not sure we want to be sending the full signed blobs in the discovery phase... many of them both agents probably have already, so it's a lot of network load.

We might need to do something like `Vec<(Arc<KitsuneAgent>, u64)>` where u64 is the signed at time - alternately, we could do a blake2b hash of the agent_info (the portion that is signed) - but might need to wait on meister's PR that makes that a Vec<u8> for consistency.